src/vl-tabs.js

```diff
@@ -146,9 +153,28 @@ export class VlTabs extends vlElement(HTMLElement) {
     }
   }
 
+  _moveTabPaneTitleSlot(tabPane) {
+    const tabPaneTitleSlot = tabPane.querySelector(`[slot="${tabPane.id}-title-slot"]`);
+    if (tabPaneTitleSlot) {
+      const clonedTabPaneTitleSlot = tabPaneTitleSlot.cloneNode(true);
+
+      const existingTabPaneTitleSlot = this.querySelector(`vl-tabs > [slot="${tabPane.id}-title-slot"]`);
+      if (existingTabPaneTitleSlot) {
+        this.replaceChild(clonedTabPaneTitleSlot, existingTabPaneTitleSlot);
+      } else {
+        this.appendChild(clonedTabPaneTitleSlot);
+      }
+
+      // We verwijderen het src title slot, anders zal de innerText van de tabsection deze nog bevatten
+      tabPaneTitleSlot.remove();
```

Comment:
Deze code lijkt me overbodig? De code zorgt er ook voor dat afnemers nooit het slot element kunnen wijzigen.

Reply:
ja zoals verduidelijkt in de comment, dit is bewust gedaan om een side-effect waarbij je de innerText van het tabSection slot bevraagd de title  text niet mee te krijgen ( zie comment op de test code, ' test/unit/vl-tabs.test.html'). Nu als dit geen probleem is kan dit voor mij weg.

Reply:
tab pane title slot wordt niet meer verwijdert